examples/Evaporation/stationary/components_3c.xml

```diff
@@ -7,7 +7,7 @@
 			<mass>1.0</mass>
 			<sigma>1.0</sigma>
 			<epsilon>1.0</epsilon>
-			<shifted>0.097901347</shifted>
+			<shifted>1</shifted>
```

Comment:
Does `true` / `false` also work here? Would be significantly more intuitive.

Reply:
yes it'll work, I'll update all the files, it'll be neater